seq2science/rules/alignment.smk

```diff
@@ -48,8 +48,9 @@ if config["aligner"] == "bowtie2":
             pipe(expand("{result_dir}/{aligner}/{{assembly}}-{{sample}}.samtools-coordinate.pipe", **config)[0]),
         log:
             expand("{log_dir}/bowtie2_align/{{assembly}}-{{sample}}.log", **config),
-        group:
-            "alignment"
```

Comment:
what's with `group: alignment` being removed?

Reply:
Originally this was necessary for rules to be executed at the same time. But since we use pipes it is not necessary, and seems to do nothing anymore. So I decided to just remove it and replace with message